src/ControlKnob.vue

```diff
@@ -254,7 +267,8 @@ watch(
   (element, oldElement) => {
     if (element && !oldElement) {
       element.addEventListener('mousedown', downListener)
-      element.addEventListener('wheel', wheelListener)
+      // Note: The wheel event is handeled according to the option, possibly manipulating scrolling behaviour
```

Comment:
While this is a good and descriptive comment, I think it's obvious for everyone reading the code that the event is handled by the options. Remove this comment.